webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleAdvancedFilterDrawer.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <exo-drawer
+      ref="peopleAdvancedFilterDrawer"
+      id="peopleAdvancedFilterDrawer"
+      right
+      @closed="close">
+    <template slot="title">
+      <span class="popupTitle">{{ $t('pepole.advanced.filter.title') }}</span>
+    </template>
+    <template slot="content">
+      <v-form class="pa-2 ms-2 mt-4">
+      <div class="d-flex flex-column flex-grow-1">
+      <div class="d-flex flex-row" v-for="item in settings" :key="item">
```

Comment:
Use `npm run lint` to format the code, this should add a single line per attribute when an element has more than 2 attributes.
Besides, you should follow this best practice for ordering the attributes, see https://eslint.vuejs.org/rules/attributes-order.html for more details.

In addition, the `:key` must be a comparable value, not an object. Use an id, name ...

```suggestion
      <div
        v-for="item in settings"
        :key="item"
        class="d-flex flex-row">
```

Reply:
fixed